src/core/EdFi.DataManagementService.Core/Extraction/DescriptorExtractor.cs

```diff
@@ -38,24 +38,31 @@ ILogger logger
                 continue;
 
             // Extract the descriptor URIs from the document
-            string[] descriptorUris = documentBody
-                .SelectNodesFromArrayPathCoerceToStrings(documentPath.Path.Value, logger)
+            var descriptorUrisWithPath = documentBody
```

Comment:
Please don't remove type information by switching to `var`. It becomes harder for me to read code in the GitHub diff. I shouldn't need to pull down a branch and view in VS Code just to understand small changes.